### ISSUE TITLE
i18n: not found zh-CN

### ISSUE THREAD

**Issue by yfdyh000** [reddit/reddit#1257]
https://crowdin.com/project/reddit/zh-CN
https://www.reddit.com/prefs/ only seen 'zh', the 'zh' seem is zh-tw (Precisely, zh-hant), zh-CN (zh-hans) is not found.
https://github.com/reddit/reddit-i18n/tree/master/reddit_i18n only 'zh'.

**Comment by yfdyh000** [comment]
@Deimos

**Comment by pawl** [comment]
@Deimos do you have a script that you're using to pull from crowdin?

I ended up using a bash script to pull from crowdin, unzip, and rename the unzipped folders names to something that works (example: zh_Hant_TW, zh_Hans_CN) : https://github.com/flask-admin/flask-admin/blob/master/babel/crowdin_pull.sh

I can make something less hacky if you're interested.

**Comment by Deimos** [comment]
@pawl: Ah, Crowdin actually supplies a CLI tool that I use: https://crowdin.com/page/cli-tool

**Comment by pawl** [comment]
@Deimos can you put a copy of the configuration you're using with the CLI tool into the reddit-i18n project (with a placeholder for the API key)?

We might need to fix something in the languages_mapping section to fix this issue.

**Comment by Deimos** [comment]
Yeah, I can look at updating the repo with the crowdin-related stuff soon. Sorry, I didn't realize that anyone other than us was actually using the i18n plugin, or I would have done that a long time ago.

In the meantime though, here's my crowdin.yaml file for the CLI tool: https://gist.github.com/Deimos/3b101cc2b83b0be86cc0

**Comment by yfdyh000** [comment]
Is there any progress?

**Comment by yfdyh000** [comment]
zh-cn appear now, but 'zh' should be rename to 'zh-TW' in github and website, and zh-cn to zh-CN for standard.
